### PR TITLE
Multi namespaces Support

### PR DESCRIPTION
**Pull request by nitishkumar71**
It will help in closing #61

### CODE REVIEW

File: tests/main_test.go [hunk 1]
```diff
@@ -91,4 +100,15 @@ type Config struct {
...
+
+	// Namespaces to verfiy OpenFaaS provider
+	Namespaces []string
+}
+
+func FromEnv(config *Config) {
+	// read CERTIFIER_NAMESPACES variable, parse as csv string
+	namespaces, present := os.LookupEnv("CERTIFIER_NAMESPACES")
+	if present {
+		config.Namespaces = strings.Split(namespaces, ",")
```
Comment (LucasRoesler) [inline tests/main_test.go:112]: I think the only improvement we could make here is to trim the space from each namespace found

https://golang.org/pkg/strings/#TrimSpace

**Review by LucasRoesler — COMMENTED** [review]
I'm sorry it took so long to get to this. Work has been a bit crazy this week.

Overall, it looks good, we can make one tiny improvement though

I am off from work this week, so I will definitely respond much more quickly.

File: tests/main_test.go (hunk 1 above)
Comment (nitishkumar71) [reply tests/main_test.go:112]: Thanks, made the changes

**Comment by nitishkumar71** [comment]
> I'm sorry it took so long to get to this. Work has been a bit crazy this week.
> 
> Overall, it looks good, we can make one tiny improvement though
> 
> I am off from work this week, so I will definitely respond much more quickly.

I can relate with this for me.

File: tests/main_test.go (hunk 1 above)
Comment (LucasRoesler) [reply tests/main_test.go:112]: Know what, let's merge this and we can handle this as we go

**Review by LucasRoesler — APPROVED** [review]

File: tests/main_test.go (hunk 1 above)
Comment (LucasRoesler) [reply tests/main_test.go:112]: lol, github was showing me a stale diff earlier, just saw the change is there :muscle:

**Pull request merged**